setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 100
```

Comment:
We might want to check if we can put this in the pyproject.toml.

Reply:
👍 Updated flake8 commands to specify pyproject.toml as config file

Reply:
Actually, flake8 wouldn't recognize the config gile, so I switched to having black specify setup.cfg for the config file